WordPress/Sniffs/WhiteSpace/ControlStructureSpacingSniff.php

```diff
@@ -289,17 +307,36 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 			}
 		}
 
-		if ( T_WHITESPACE !== $this->tokens[ ( $parenthesisOpener + 1) ]['code']
-			&& T_CLOSE_PARENTHESIS !== $this->tokens[ ( $parenthesisOpener + 1) ]['code']
-		) {
-			// Checking this: $value = my_function([*]...).
-			$error = 'No space after opening parenthesis is prohibited';
-			$fix   = $phpcsFile->addFixableError( $error, $stackPtr, 'NoSpaceAfterOpenParenthesis' );
+		if ( T_CLOSE_PARENTHESIS !== $this->tokens[ ( $parenthesisOpener + 1) ]['code'] ) {
+			if ( T_WHITESPACE !== $this->tokens[ ( $parenthesisOpener + 1) ]['code'] ) {
```

Comment:
Well spotted! Now why isn't there a sniff enabled to notify me about that ? :sunglasses: (Fixed)